js/code.js

```diff
@@ -114,10 +114,10 @@ function addColor()
 	let newColor = document.getElementById("colorText").value;
 	document.getElementById("colorAddResult").innerHTML = "";
 
-	let tmp = {color:newColor,userId,userId};
+	let tmp = {user:1, firstName:newColor, lastName:"Parker", email:"1@yahoo.com", phone:3333333333 };

```

Comment:
This temporarily allows to search for contacts while using place holder values for future development